### PR TITLE
[ticket/13873] Removed broken print stylesheet.

### PR DESCRIPTION
**Pull request by callumacrae**
I also made the other stylesheets work on all devices. In addition to being effort to fix, the print stylesheet was hurting performance.

https://tracker.phpbb.com/browse/PHPBB3-13873

### CODE REVIEW

File: phpBB/styles/prosilver/template/overall_header.html [hunk 1]
```diff
@@ -27,22 +27,21 @@
 	Modified by:
 -->
 
-<link href="{T_THEME_PATH}/print.css?assets_version={T_ASSETS_VERSION}" rel="stylesheet" type="text/css" media="print" title="printonly" />
-<!-- IF S_ALLOW_CDN --><link href="//fonts.googleapis.com/css?family=Open+Sans:600&amp;subset=latin,cyrillic-ext,latin-ext,cyrillic,greek-ext,greek,vietnamese" rel="stylesheet" type="text/css" media="screen, projection" /><!-- ENDIF -->
-<link href="{T_STYLESHEET_LINK}" rel="stylesheet" type="text/css" media="screen, projection" />
-<link href="{T_STYLESHEET_LANG_LINK}" rel="stylesheet" type="text/css" media="screen, projection" />
-<link href="{T_THEME_PATH}/responsive.css?assets_version={T_ASSETS_VERSION}" rel="stylesheet" type="text/css" media="only screen and (max-width: 700px), only screen and (max-device-width: 700px)" />
+<!-- IF S_ALLOW_CDN --><link href="//fonts.googleapis.com/css?family=Open+Sans:600&amp;subset=latin,cyrillic-ext,latin-ext,cyrillic,greek-ext,greek,vietnamese" rel="stylesheet"><!-- ENDIF -->
+<link href="{T_STYLESHEET_LINK}" rel="stylesheet">
```
Comment (gooof) [inline phpBB/styles/prosilver/template/overall_header.html]: type="text/css" ?

Comment (callumacrae) [inline phpBB/styles/prosilver/template/overall_header.html]: optional in HTML5, therefore redundant

**Comment by callumacrae** [comment]
@Derky To be fair, it was completely broken before

screen shot 2015-05-28 at 14 37 27

**Comment by Derky** [comment]
@callumacrae but the same stylesheet is still being used in viewtopic_print.html https://github.com/callumacrae/phpbb/blob/ticket/13873/phpBB/styles/prosilver/template/viewtopic_print.html#L9 
So shouldn't you fix that stylesheet instead?

**Comment by callumacrae** [comment]
@Derky The stylesheet is fine, the print view works.

**Comment by marc1706** [comment]
The viewtopic print view is not broken. Using the browser's print view was very much broken. Now it's slightly less broken.

**Pull request merged**